installer/build-server.py

```diff
@@ -266,10 +266,10 @@ def parse_version_info(version_path):
             github_release = get_release(repo, "daily")
             needs_upload = False
 
-    # Output git desription
+    # Output git description
     output("git description of openshot-qt-git: %s" % openshot_qt_git_desc)
 
-    # Output git desription
+    # Output git description
```

Comment:
any reason why lines 269 and 272 are exactly the same ?

Reply:
Copy and paste? :)

Reply:
@DylanC I was just wondering if there was a redundancy issue redundancy :wink:

Reply:
There's plenty of copy-paste comment duplication (not to mention _wrong_ comments, because the comments weren't edited after being copied, but the code was) in the code.

Reply:
The bigger question is why lines 270 and 273 are the same. Because it's not just the comment that appears twice, it's the _code_. Which means it outputs the same line twice for absolutely no reason.

Reply:
right, @ferdnyc shall I remove one of them ?

Reply:
*shrug* Your call. It's the build script, not part of the OpenShot source, so it really only affects @jonoomph . But I can't see any reason that would be there twice, other than pure accident.